### ISSUE TITLE
Application unable to start correctly. (0xc0000142)

### ISSUE THREAD

**Issue by Hefaustas** [pawREP/ZHM5Randomizer#17]
When I try to launch the I get the following error message:
![image](https://user-images.githubusercontent.com/54642418/104240495-d1e76e00-5464-11eb-9e42-8bce0c83a664.png)
Removing all of this mod's files fix the issue. 
I also tried running the game from the DX12 executable. The game would launch, and everything would work fine, but the randomizer itself wouldn't work.

**Comment by Mahdix6x9x** [comment]
When I install version 1.33 above it says the randomizer doesn't support the game version,
I used the 1.32 version below and it works but at the start it has the 0x00142 error again.
I don't know what do do 
I have steam and I have added the game and launched it but it doesn't work again because my steam is offline supported.
I think my pc needs a special software or something special. Anyone has any IDEA?
